autoexport: Add support for console exporters (#4486)
open-telemetry/opentelemetry-go-contrib: CHANGELOG.md

- Add `"go.opentelemetry.io/contrib/config"` package that includes configuration models generated via go-jsonschema. (#4376)
- Add `NewSDK` function to `"go.opentelemetry.io/contrib/config"`. The initial implementation only returns noop providers. (#4414)
- Add metrics support (No-op, OTLP and Prometheus) to `go.opentelemetry.io/contrib/exporters/autoexport`. (#4229, #4479)

> - Add support for standard output exporter via environment variables for the metric and trace signals identified by the key: `console`. (#4486)

Reviewer: ```suggestion
- Add support for `console` span exporter and metrics exporter in `go.opentelemetry.io/contrib/exporters/autoexport`. (#4486)
```
